designer/server/src/routes/forms/library.js

```diff
@@ -48,7 +55,19 @@ export default [
         }
       },
       validate: {
-        query: paginationOptionsSchema
+        query: Joi.object(paginationOptionFields).keys({
+          sort: Joi.string()
+            .valid('updatedDesc', 'updatedAsc', 'titleAsc', 'titleDesc')
+            .optional()
+        }),
+        failAction: (request, _h, error) => {
+          request.log('error', {
+            message: error?.message,
+            stack: error?.stack
+          })
+
+          throw Boom.badRequest()
```

Comment:
Added `Boom.badRequest()` error for validation errors- this will redirect to the page @alexluckett [just merged in](https://github.com/DEFRA/forms-designer/pull/627).